quickstarts/temporal-cloud/config.yml

```diff
@@ -0,0 +1,41 @@
+slug: temporal-cloud
+description: |
+  ## Why monitor Temporal cloud?
+  Monitoring allows you to identify performance bottlenecks, such as slow workflows or inefficient resource usage, optimize your Temporal cloud for better performance, and help you detect and diagnose faults or errors in your Temporal workflows.
+
+  ## Comprehensive monitoring quickstart for Temporal cloud
+  Temporal cloud provides users with workflow execution states and metadata for debugging purposes. Integrating Temporal cloud with New Relic enhances system health maintenance, early issue identification, and smooth operation assurance. This integration leverages the strengths of both systems to offer a comprehensive monitoring solution, encompassing metrics and alerts. Customize the following steps based on the specific programming language and framework of your Temporal cloud services.
+
+  ## What’s included in this quickstart?
+  New Relic Temporal cloud monitoring quickstart ability to cover quality on out-of-the-box reporting.
+
+  - Dashboards (failed, cancel and terminated workflows, service latency and frontend service requests)
+  - Alerts (CPU utilization, failed workflows and service latency)
```

Comment:
```suggestion
  - Alerts (failed workflows and service latency)
```

Reply:
Hi @sjyothi54 ,
I have updated the review changes.